### PR TITLE
Project 3: Using Bootstrap - Milestone 1 

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
## In this project we: 
- Installed Bootstrap in our project.
- Made 3 breakpoints using Bootstrap classes( mobile, tablet, and desktop).
- Created Header and Footer for Newsweek clone page in these 3 sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,15 +7,152 @@
...
   <title>Newsweek-project</title>
 </head>
-
 <body>
+  <div class="container-lg container-fluid mt-5">
+    
+      <nav class="navbar navbar-expand-lg navbar-light bg-danger d-flex  flex-lg-column pb-0">
+        <div class="container-fluid d-flex">
+          <time class="text-light d-none d-lg-inline" datetime="2021-01-20">Wed, Jan 20, 2021</time>
+          <img class="logo" src="./img/Newsweek_Logo.png" alt="Newsweek logo">
```
Comment (teekaytech) [inline index.html]: This should be a link. i.e. wrap the `<a>` tag around it, with `#` as the `href` attribute value.

File: index.html [hunk 2]
```diff
@@ -7,15 +7,152 @@
...
+                <a class="nav-link" href="#"><b>STIMULUS PACKAGE</b> </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b>VACCINE</b> </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b>CAPITOL RIOTS</b> </a>
+              </li>
+            </ul>
+          </div>
```
Comment (teekaytech) [inline index.html]: You should be more semantic here: use the `<nav>` tag for the main navigation menu.

File: index.html [hunk 3]
```diff
@@ -7,15 +7,152 @@
...
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b>Vantage</b> </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link disabled" href="#" ><b>Search</b><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
+                  <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
+                </svg> </a>
+                
+              </li>
```
Comment (teekaytech) [inline index.html]: This item should be a form with the search field, not just an icon or text

File: index.html [hunk 4]
```diff
@@ -7,15 +7,152 @@
...
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b> The Debate</b></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b>Experts</b> </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"><b>Vantage</b> </a>
+              </li>
```
Comment (teekaytech) [inline index.html]: Use the `<nav>` tag for the main navigation menu.

File: index.html [hunk 5]
```diff
@@ -7,15 +7,152 @@
...
+            </ul>
+          </div>
+        
+      </nav>
+    
+  </div>
+  <footer class="container-fluid container-lg mt-5">
+    <div class="footer-top d-flex flex-column justify-content-between align-items-start flex-lg-row">
+      <div class="wrap-left d-flex flex-column flex-lg-row ">
+        <img class="pb-3 me-3" src="img/Newsweek_logo_transparent.png" alt=" Newsweek logo">
```
Comment (teekaytech) [inline index.html]: The Newsweek logo at the footer section should also be a link.

File: index.html [hunk 6]
```diff
@@ -7,15 +7,152 @@
...
+        <p class="mb-2">&copy;2021   NEWSWEEK DIGITAL LLC</p>
+      </div>
+      <div class="social  mb-3">
+        <i class="fab fa-facebook-square me-2"></i>
+        <i class="fab fa-twitter me-2"></i>
+        <i class="fab fa-instagram me-2"></i>
+        <i class="fab fa-tumblr me-2"></i>
+        <i class="fab fa-linkedin-in"></i>
```
Comment (teekaytech) [inline index.html]: The social media icons should be linked also.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @ShinobiWarior & @emmyobonyo :wave:

__You've really done well so far. Here're the major highlights of this PR:__

:+1:  Nice PR title and description
:+1: Linters running/passing successfully
:+1: Extensive usage of bootstrap classes
:+1: Your responsiveness looks promising :wink: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_**Note: You are asked to be more precise and to make sure that your page looks exactly the same as the source page**_

- [x] Update your Readme file: please add project description, getting started, live demo, and other necessary contents to your Readme.
- [x] You should make the Newsweek on the top navbar bigger. You should also add padding to the container the top Newsweek logo.
- [x] Observe the difference between your footer main navigation menu and the original one. Kindly add some space (padding, margin) between each of the nav items.
Original page:
original
Your clone page:
clone

- [x] Check my inline comments below and fix them as appropriate.

**[ OPTIONAL CHANGES ]**

- Using the `!important` for the top banner background image might not be important. Instead of trying to manipulate the bootstrap class, you can create your own class and add it to the list of other bootstrap classes for your element.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
#### Excellent job on this project @ShinobiWarior  and @emmyobonyo. :fireworks: Keep it up.  :champagne: :champagne: **All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete. 👍**

### **Kindly have a look at the following OPTIONAL comments below.**
 

- [ ] On 360px wide smart phones. your header is a bit messed up. Check the screenshot below and look into it.
### Original 👇 
image

### Yours 👇 
image

I wish you all the very best in your next tasks:1st_place_medal:
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Pull request merged**